mixer/cmd/init.go

```diff
@@ -80,6 +82,7 @@ func init() {
 	initCmd.Flags().StringVar(&initFlags.clearVer, "upstream-version", "latest", "Alias to --clear-version")
 	initCmd.Flags().IntVar(&initFlags.mixver, "mix-version", 10, "Supply the Mix version to build")
 	initCmd.Flags().StringVar(&initFlags.upstreamURL, "upstream-url", "https://cdn.download.clearlinux.org", "Supply an upstream URL to use for mixing")
+	initCmd.Flags().StringVar(&initFlags.bundlesURL, "upstream-bundles-url", builder.UpstreamBundlesURL, "Supply an upstream bundles URL to get bundle definitions")
```

Comment:
Default should be `""`. This will make sure the default set in config.go is used. We only need values here for things that are not inside `config/`.  See `format` below as a reference.